Makefile

```diff
@@ -54,12 +54,16 @@ DOCKER_ENVS := \
 BIND_DIR := $(if $(BINDDIR),$(BINDDIR),$(if $(DOCKER_HOST),,bundles))
 DOCKER_MOUNT := $(if $(BIND_DIR),-v "$(CURDIR)/$(BIND_DIR):/go/src/github.com/docker/docker/$(BIND_DIR)")
 
+# This allows the test suite to be able to run without worrying about the underlying fs used by the container running the daemon (e.g. aufs-on-aufs), so long as the host running the container is running a supported fs.
+# The volume will be cleaned up when the container is removed due to `--rm`.
+DOCKER_BUNDLES_VOLUME := -v /go/src/github.com/docker/docker/bundles
```

Comment:
I like the idea, but this will definitely interact in strange ways with `DOCKER_MOUNT` above, won't it? (not to mention user expectations that built artifacts are available locally in `bundles`, which is how it's worked for a very long time now :disappointed:)